x-pack/solutions/observability/plugins/observability/public/pages/overview/components/sections/empty/empty_sections.tsx

```diff
@@ -47,6 +48,7 @@ export function EmptySections() {
               key={app.id}
               style={{
                 border: `${theme.eui.euiBorderEditable}`,
+                borderColor: highContrastMode ? euiTheme.border.color : '',
```

Comment:
```suggestion
                ...(highContrastMode ? { borderColor: euiTheme.border.color } : {}),
```

what do you think about this change to avoid having `borderColor: ''`?

Reply:
Oh yes, great suggestion! This actually currently results in the wrong color because it unsets the border color when using default `style`. I'm too used to using Emotions `css` apparently that handles it automatically 🤦‍♀️ 

Fyi, the usage of `style` here should eventually be migrated to use Emotions `css`.

Reply:
Updated in this [commit](https://github.com/elastic/kibana/pull/216964/commits/7f3f089112035f0dd8d866eafa24bf169c73d906).

Reply:
@rmyz Actually I realized we can just simply set the border color as the token is flipped automatically in high contrast mode. no need for the check.